docs/userguides/accounts.md

```diff
@@ -134,6 +137,25 @@ ape accounts generate <ALIAS> --word-count <WORDCOUNT>
 
 If you do not use the `--word-count` option, Ape will use the default word count of 12.
 You can use all of these together or separately to control the way Ape creates and displays your account information.
+
+This same functionality is also scriptable with the same inputs as the `generate` command:
+
+```python
+from ape_accounts import generate_account
+
+alias = "my-account"
+passphrase = "mySecurePassphrase"
+hd_path = "m/44'/60'/0'/0/0"
+word_count = 12
+
+account, mnemonic = generate_account(alias, passphrase, hd_path, word_count)
```

Comment:
Now that we are linking to the methoddocs, I think this is appropiate to drop the hd path/word count
```suggestion
account, mnemonic = generate_account(alias, passphrase)
```